Fabric.Identity.API/scripts/tests/identity-cli.tests.ps1

```diff
@@ -71,4 +71,69 @@ describe 'Get-FabricInstallerAccessToken' {
             }
         }
     }
-}
\ No newline at end of file
+}
+
+describe 'Get-ClientRegistration' {
+    Context 'Unit Tests' {
+        Context 'Valid Request' {
+            It 'Should return a client object when valid response' {
+                $mockUrl = New-MockObject -Type Uri
+                $allowedGrantTypes = @("client_credentials", "delegation")
+                $allowedScopes = @("fabric/authorization.read", "dos/metadata", "fabric.profile")
```

Comment:
I assume these are leftover, did it not like trying to use these inside of the mock either?

Reply:
good catch